Makefile

```diff
@@ -170,23 +178,26 @@ vendor_libs: $(BACKBONE_OBJS) $(CLASSNAME_OBJS) $(LODASH_OBJS) $(REACT_OBJS) \
              $(BUILT)/test/vendor/mocha.js $(BUILT)/test/vendor/mocha.css \
              $(BUILT)/test/vendor/chai.js $(BUILT)/test/vendor/chai-as-promised.js
 
-$(BUILT)/add-on/chrome/content/preferences/prefs.js: $(VENV) add-on/preferences/prefs.js
+$(BUILT)/add-on/chrome/content/preferences/prefs.js: add-on/preferences/prefs.js $(VENV) Makefile
 	mkdir -p $(@D)
 	. $(VENV)/bin/activate && \
 	  python $(VENV)/lib/python2.7/site-packages/mozbuild/preprocessor.py \
-	         -D DEBUG=1 -D LOOP_BETA=1 -o $@ add-on/preferences/prefs.js
+	         -D DEBUG=1 -D LOOP_BETA=1 $(LOOP_DEV_XPI) \
+	         -o $@ $<
 
-$(DIST)/add-on/chrome/content/preferences/prefs.js: $(VENV) add-on/preferences/prefs.js
+$(DIST)/add-on/chrome/content/preferences/prefs.js: add-on/preferences/prefs.js $(VENV) Makefile
```

Comment:
So I'm wondering if this  change wants to be made to the add-on_dist target, because doing a "make dist_xpi" doesn't produce a development XPI.  Or is there some piece of the machinery that has yet to be landed?